results/pkg/api/server/v1alpha1/server.go

```diff
@@ -401,6 +419,18 @@ func (s Server) getResultByID(name string) (*pb.Result, error) {
 	return result, nil
 }
 
+func getResultETag(result *pb.Result) (etag string, err error) {
```

Comment:
Even though it's not an exported func, let's add godoc to explain what this is doing so it's clear when you want to use this vs `result.GetEtag()`